src/controllers/DeleteProfileController.ts

```diff
@@ -0,0 +1,20 @@
+import { FileSystemProfilePersistenceService } from "../infrastructure/file-system/FileSystemProfilePersistenceService";
+
+export class DeleteProfileController {
+  private persistenceService: FileSystemProfilePersistenceService;
+
+  constructor() {
+    this.persistenceService = new FileSystemProfilePersistenceService();
+  }
+
+  public control(name: string, confirmation: boolean): boolean {
+    if (confirmation) {
+      const profile = this.persistenceService.find(name);
```

Comment:
If you merge main into this branch you have, getProfile controller, you can use it, to find profiles. Same as discused in class, for the editController